<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogHomeSrch</class>
 <widget class="QDialog" name="DialogHomeSrch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>282</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Home and Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="cbHomeSearch">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>HOME</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SRCH</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbHsOptions"/>
       </item>
       <item>
        <widget class="QComboBox" name="cbEdge">
         <item>
          <property name="text">
           <string>POSEDGE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NEGEDGE</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbDirection">
         <item>
          <property name="text">
           <string>-1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>+1</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnGoStop">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frameValues">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1">
         <widget class="QLCDNumber" name="lcdPosEncin">
          <property name="digitCount">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLCDNumber" name="lcdEncTgtenc">
          <property name="digitCount">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLCDNumber" name="lcdEncAbsenc">
          <property name="digitCount">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLCDNumber" name="lcdPosAbsenc">
          <property name="digitCount">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLCDNumber" name="lcdPosInpos">
          <property name="digitCount">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLCDNumber" name="lcdEncEncin">
          <property name="digitCount">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLCDNumber" name="lcdEncAxis">
          <property name="digitCount">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLCDNumber" name="lcdEncInpos">
          <property name="digitCount">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelEnc">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>ENC</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelPos">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>POS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLCDNumber" name="lcdPosTgtenc">
          <property name="digitCount">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLCDNumber" name="lcdPosAxis">
          <property name="digitCount">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="5" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelAbsenc">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>ABSENC</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelInpos">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>INPOS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelEncin">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>ENCIN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelTgtenc">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string> TGTENC </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="labelAxis">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>AXIS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGraphicsView" name="gvIndicator">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bbClose">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bbClose</sender>
   <signal>rejected()</signal>
   <receiver>DialogHomeSrch</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbClose</sender>
   <signal>accepted()</signal>
   <receiver>DialogHomeSrch</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
